nemo_skills/evaluation/evaluator/segale_judge.py

```
@@ -0,0 +1,1145 @@
+        span["comet-qe"] = comet_qe_scores[idx]
+        span["metricx-qe"] = metricx_qe_scores[idx]
+
+    # Compute language fidelity per (doc_id, lang) from original records.
+    orig_by_key: dict[tuple[str, str], dict] = {}
+    for r in orig_records:
+        if "doc_id" in r:
+            orig_by_key[(r["doc_id"], r.get("target_language", ""))] = r
+
+    doc_lang_fidelity: dict[tuple[str, str], float] = {}
+    seen_pairs: set[tuple[str, str]] = set()
+    for span, lang in zip(all_spans, all_span_langs):
```

Ruff (0.15.9)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)